### ISSUE TITLE
Calendar forgots its link to my email account

### ISSUE THREAD

**Issue by xhpohanka** [jobisoft/EAS-4-TbSync#208]
## Your environment

TbSync version: 4.3
EAS-4-TbSync version: 4.1.5
Thunderbird version: 102.4.2

I have an email account in Thunderbird created with Exquilla (no other option in our company, OWA is forbidden). 
For calendar I use synchronization using TB sync and exchange provider.
Unfortunately the link between calendar and account is lost by Thunderbird. It can be set in calendar properties, but it disappears when properties are reopen.
The only way to fix I find is to set calendar as read only and then again as writable. But it works only until thunderbird is restarted...

I'd be grateful if you can help somehow.

**Comment by tgoeg** [comment]
Same setup, seeing this as well.
Can we provide any debug logs or similar resources to help in drilling down to the core issue?

**Comment by jobisoft** [comment]
1) Is the issue limited to Exquilla accounts, or is the setting also lost for TbSync calendars connected to standard IMAP accounts?
2) Is this issue limited to TbSync calendars, or is the setting also lost for standard Thunderbird calendars connected to 
 2a) Exquilla accounts
 2b) standard IMAP accounts?

**Comment by xhpohanka** [comment]
I will try to answer your questions later, but I noticed that if I have another calendars from the same tbsync account (tasks and birthdays) then they are used instead of the first main calendar account...

**Comment by tgoeg** [comment]
All the IMAP accounts I have do not support calendars, it seems, they all offer calendars via calDAV.

I tried with one of those calDAV calendars with Thunderbird's built in calDAV support, and in this case, the mail address sticks successfully.

I also tried with a calDAV calendar and Provider for CalDAV & CardDAV and TB sync, and it also works. I think this answers 1) and 2b), but maybe IMAP is a mandatory criteria.

2a) does not seem to be possible (at least in my case), as Exquilla alone does not enable calendars to be configured with vanilla Thunderbird, I need to take the ActiveSync route to actually get any calendar sync.

**Comment by xhpohanka** [comment]
2a) local offline calendar seems to work with exquilla account as expected, it keeps its link to email across restarts

**Comment by tgoeg** [comment]
Yes, I can confirm it has worked with Thunderbird 91.13.0 (and the according add-on versions) before, as well.
Debian stable's Thunderbird 102.3.0 does not work with the new add-ons, which is a little impractical. I reverted to vanilla Thunderbird 102.4.x directly from thunderbird.net.

**Comment by jobisoft** [comment]
The project has moved on to a total rewrite, the new official and only supported version on GitHub is v5:

https://github.com/jobisoft/TbSync/releases
https://github.com/jobisoft/EAS-4-TbSync/releases

If you can reproduce the issue on v5, please create a new issue with some debug info. Thank you for your support.
